main.js

```diff
@@ -285,7 +285,7 @@ async function processMessage(obj) {
                 break;
             }
             case 'getAnnouncementSounds': {
-                let files = await libs.fs.readdirSync(`${__dirname.replace('node_modules/iobroker.sayit', '')}/iobroker-data/files/${adapter.namespace}/tts.userfiles`);
+                let files = await adapter.readDirAsync('sayit', `${__dirname.replace('node_modules/iobroker.sayit', '')}/iobroker-data/files/${adapter.namespace}/tts.userfiles`);
```

Comment:
The call will not work that way!
The first parameter should (!!) be ${adapter.namespace} and second parameter is then "tts.userfiles" ... but check it ... all depends how themeta objects are created